telemeta/migrations/0035_auto__chg_field_recordingcontext_notes__chg_field_recordingcontext_val.py

```diff
@@ -339,8 +339,7 @@ def forwards(self, orm):
         # Changing field 'MediaItem.comment'
         db.alter_column('media_items', 'comment', self.gf('telemeta.models.fields.TextField')())
 
-        # Changing field 'MediaItem.code'
-        db.alter_column('media_items', 'code', self.gf('telemeta.models.fields.CharField')(unique=True, max_length=250))
```

Comment:
@thomasfillon can you tell why this was indeed a blocking altering on the CREM's db (I think 0035 was yours) ?